Topic-4-Javascript-Desing-Patterns/1/1_A.js

```diff
@@ -0,0 +1,64 @@
+//RENAME FILE BEFORE COMMIT
+class Movie {
+
+    constructor (title,year,duration) {
+        this.title = title;
+        this.year = year;
+        this.duration = duration;
+    }
```

Comment:
Maybe you could add the name of the movie in the console.log to be more descriptive